jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -318,6 +352,7 @@ YUI.add('user-profile-entity', function() {
           name: name,
           owner: entity.owner
         });
+        destroyButton = this._generateDestroyButton(entity);
```

Comment:
because the `_generateDestroyButton` returns `null` could you not just put this method call on line 378 directly?